File: SoloLearn/Duct Tape/DuctTape.java
import java.util.Scanner;

public class DuctTape {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int height, width, area;
        height = input.nextInt();
        input.nextLine();
        width = input.nextInt();
        area = height * width * 2;
        System.out.println((int)Math.ceil(area / 10.0));
        input.close();
    }
} 

File: SoloLearn/Duct Tape/DuctTape.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int height = Convert.ToInt32(Console.ReadLine()), width = Convert.ToInt32(Console.ReadLine()), area = height * width * 2;
            Console.WriteLine(Math.Ceiling(area / 10.0));
        }
    }
} 